checkout-core/src/main/java/com/adyen/checkout/core/internal/util/LocaleUtil.kt

```diff
@@ -36,7 +40,11 @@ object LocaleUtil {
      */
     @JvmStatic
     fun fromLanguageTag(tag: String): Locale {
-        return Locale.forLanguageTag(tag)
+        val locale = Locale.forLanguageTag(tag)
+        if (!isValidLocale(locale)) {
```

Comment:
Isn't this redundant? `forLanguageTag` already safely parses the tag, so I think it won't return an invalid locale

Reply:
Fair point. But I'm not sure if it's obvious that `forLanguageTag` only returns valid locales. And I'm wondering if our validation and `forLanguageTag` work the exact same way internally. Or if we'll need to change our validation in the future and forget to re-add it here.
WDYT?

Reply:
Yeah we've had this validation before and it caused issues, I think it considered some locales invalid that were actually valid.

In all cases the validation is now very basic and maybe only prevents obviously invalid use cases. But my point is that if we change it later on to something more robust and we don't call it inside `fromLanguageTag` it might cause issues.

Anyway happy to remove it if you think it's bad to have this redundancy.

Reply:
Hmm yeah that is the problem with hardcoded lists, if countries or languages change they don't reflect the truth anymore.

I don't think we will ever create a more robust validation with all this in mind, so I think it's safe to remove

Reply:
👍